stable/jenkins/values.yaml

```diff
@@ -308,7 +308,9 @@ master:
         #   requests:
         #     cpu: 50m
         #     memory: 50Mi
-      # env:
+      env:
+        - name: REQ_RETRY_CONNECT
+          value: 10
```

Comment:
@torstenwalter looks like this needs to be in quotes as CI is failing. could also update the default value in the README.

Reply:
We know in a couple of minutes if this fixes it.